docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/home_landing/paragraph--ncids-promo-block-external.html.twig

```diff
@@ -16,6 +16,8 @@
 {% set promo_theme = paragraph.field_ncids_theme.value %}
 {% if promo_theme == 'light' %}
     {% set promo_theme = '' %}
+{% elseif promo_theme == 'dark' %}
```

Comment:
Can you change this to **also** match `nci-promo-block--dark`?

The reason is that on production all the dark promoblocks already have the value `nci-promo-block--dark` in the database.

Reply:
Also create an issue to fix this once prod has been updated with the new value. Add a note this is a hack, and put in the URL of the ticket in your comment. A multiline comment in twig looks like below. (I think)
```
{# /*
  I am a comment
*/ #}
```

Reply:
I might be wrong but if database has the old value `nci-promo-block--dark`, then `promo_theme` would still be `nci-promo-block--dark` because `paragraph.field_ncids_theme.value` is neither `light` nor `dark` so the `if` and `elseif` won't be triggered, and `promo_theme` would just be `nci-promo-block--dark`. If I'm right, then this avoids the bug ticket to remove the hack all together, all we would need to do is resave the landing pages once prod is updated.